### DISCUSSION TITLE
override specified config?

### DISCUSSION THREAD

**Discussion by uyarn** [eslint/eslint#13584] [General] [answered]
I have a mix js & ts repos.
I set some rules、globals and env configuration for the .js files.
I just want to change the parser from babel-eslint to @typescript-eslint/parser in the .ts files. 
However, when I use overrides, the rules、globals and env seem to be overrided as well.
How can I preserve it? Or I need to write the rules again?
```json
{
"parser": "babel-eslint",
"rules": {
   "eqeqeq":"error" // this rule is overrided as well 
},
"overrides":[{
    "files": ["*.ts", "*.tsx"],
    "parser": "@typescript-eslint/parser",
  }]
}
```

**Comment by kaicataldo** [comment, marked as answer]
I would expect the config you've shared to do behave as you intended. The overrides should be merged into the top-level config. I wonder if there might be something else going on here. Is there any chance you could make a Git repository that we can look at that has the minimal reproduction case?

**Reply by uyarn** [reply]
hi @kaicataldo , I am sorry that the Git repository is private. well, I would check again. Thanks for your reply. Cheers~

**Reply by kaicataldo** [reply]
No problem! I tried to reproduce this on my machine and was unable to. A reproduction case would go a long way in helping us help you, if you end up stuck.
